DEVELOPMENT.md

```diff
@@ -88,6 +96,32 @@ registry which if you are developing locally you could use `export KO_DOCKER_REP
 to use the local one on kind. Please see official
 [KO documentation](https://github.com/google/ko#local-publishing-options) for more information.
 
+#### Running end-to-end tests locally
+
+**NOTE** The `dispatch/path`, `host-rewrite`, and `upgrade` tests seem to fail (as of Jan 28) when run locally on Kind. You'll want to add these to the `--skip-tests` flag in `hack/e2e-tests.sh`.
+
+Run the tests from the root directory; the script will create a _new_ Kind cluster and build and install `net-contour` on it, along with building and pushing test images.
+
+```bash
+./test/e2e-tests.sh --cloud-provider kind
+```
+
+**WHILE THE TESTS ARE RUNNING**, install MetalLB in the cluster:
```

Comment:
It seems to be **very important** (lol) to do this while the tests are running. I would move the last paragraph to immediately after this.


-> this paragraph:
> You should do this once the cluster is created, while the tests say "`Waiting until service envoy in namespace contour-external has an external address (IP/hostname)...`".

Reply:
I needed to do this while the cluster was running because the script creates the cluster and then waits for all the things (including the LoadBalancer Service) to be ready, but doesn't have a way to run an extra script for setting up the load balancer.